Templates/js/script.js

```diff
@@ -61,6 +61,151 @@ function openForm() {
 function closeForm() {
     document.getElementById("myForm").style.display = "none";
 }
-function signUp() {
-    
+// Messages 
+function message(type,message){
+	if(type=='success'){
+      var div = document.getElementById('succ');
+      div.getElementsByTagName('p')[0].innerText = message;
+      div.style.display = "block";
+       	document.getElementById("sclosebtn").onclick = ()=>{
+       	document.getElementById('succ').style.display = 'none';
+       }
+    }
+    else if(type=='danger'){
+      var div =document.getElementById('dan');
+      div.getElementsByTagName('p')[0].innerText = message;
+      div.style.display = "block";
+       	document.getElementById("dclosebtn").onclick = ()=>{
+         document.getElementById('dan').style.display = 'none';
+         }
+    }
+    else if(type=='info'){
+      var div =document.getElementById('inf');
+      div.getElementsByTagName('p')[0].innerText = message;
+      div.style.display = "block";
+       	document.getElementById("iclosebtn").onclick = ()=>{
+         document.getElementById('inf').style.display = 'none';
+         }
+    }
+    else if(type=='worning'){
```

Comment:
Fix the typo and say `warning` instead of working and change the origin of this variable too.